src/quo/components/wallet/summary_info/view.cljs

```diff
@@ -26,34 +27,47 @@
 
 (defn networks
   [values theme]
-  (let [{:keys [ethereum optimism arbitrum]} values]
+  (let [{:keys [ethereum optimism arbitrum]} values
+        show-optimism?                       (pos? optimism)
+        show-arbitrum?                       (pos? arbitrum)]
     [rn/view
      {:style               style/networks-container
       :accessibility-label :networks}
-     [network-amount
-      {:network  :ethereum
-       :amount   (str ethereum " ETH")
-       :divider? true
-       :theme    theme}]
-     [network-amount
-      {:network  :optimism
-       :amount   (str optimism " ETH")
-       :divider? true
-       :theme    theme}]
-     [network-amount
-      {:network :arbitrum
-       :amount  (str arbitrum " ETH")
-       :theme   theme}]]))
+     (when (pos? ethereum)
+       [network-amount
+        {:network  :ethereum
+         :amount   (str ethereum " ETH")
+         :divider? (or show-arbitrum? show-optimism?)
+         :theme    theme}])
+     (when show-optimism?
+       [network-amount
+        {:network  :optimism
+         :amount   (str optimism " OPT")
+         :divider? show-arbitrum?
+         :theme    theme}])
+     (when show-arbitrum?
+       [network-amount
+        {:network :arbitrum
+         :amount  (str arbitrum " ARB")
+         :theme   theme}])]))
 
 (defn- view-internal
   [{:keys [theme type account-props networks? values]}]
   [rn/view
    {:style (style/container networks? theme)}
    [rn/view
     {:style style/info-container}
-    (if (= type :status-account)
-      [account-avatar/view account-props]
-      [user-avatar/user-avatar account-props])
+    (cond (= type :status-account)
+          [account-avatar/view account-props]
+          (= type :saved-account)
+          [wallet-user-avatar/wallet-user-avatar (assoc account-props :size :size-32)]
+          (= type :account)
```

Comment:
some slight differences for designs then what was initially implemented here. I think it went unnoticed and will ask for a design review for this component.